app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.counter)
+  end
```

Comment:
![image](https://github.com/fickryiman/blog-app/assets/78251927/664b479e-a39c-40ec-abc7-f12b58497cab)

Please fix the error in the `Like model` to update the likes counter for a post. 🛠🤗

Reply:
OK, there is a typo, it should be post.likes.count instead of post.likes.counter

Reply:
![image](https://github.com/fickryiman/blog-app/assets/116701353/76fe26cd-35f1-49f8-b88e-42758bbe58f2)